app/frontend/js/form-header-submission.js

```diff
@@ -0,0 +1,39 @@
+const contentSourceFormSubmit = document.getElementById('js-content-source-form-submit');
+
+if(contentSourceFormSubmit) {
+  contentSourceFormSubmit.addEventListener("click", (event) => {
+    document.getElementById('js-content-source-form').submit();
+  });
+}
+
+const extractionFormSubmit = document.getElementById('js-extraction-definition-form-submit');
```

Comment:
Looks like lots of dulication - code climate might not be happy

Reply:
Fair call! Iv updated this file to be dynamic and all you need now is an id on your form and a data-form attribute on your button.